seed/static/seed/js/services/meter_service.js

```diff
@@ -4,10 +4,11 @@ angular.module('BE.seed.service.meter', [])
     function ($http) {
       var meter_factory = {};
 
-      meter_factory.get_meters = function (property_view_id) {
```

Comment:
Nice - I could see arguments for moving these to properties, but I think it's better to have a meters service. Otherwise the inventory service would basically have everything in there.